packages/python3.rb

```diff
@@ -130,13 +132,20 @@ def self.check
   def self.install
     Dir.chdir 'builddir' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+      system "#{CREW_DEST_PREFIX}/bin/python3 -m ensurepip --upgrade --default-pip"
+      system "sed -i 's,#{CREW_DEST_PREFIX}/bin/python3,#{CREW_PREFIX}/bin/python3,g' #{CREW_DEST_PREFIX}/bin/pip3"
     end
 
     # Make python3 the default python
     FileUtils.ln_sf 'python3', "#{CREW_DEST_PREFIX}/bin/python"
+    FileUtils.ln_sf 'pip3', "#{CREW_DEST_PREFIX}/bin/pip"
   end
 
   def self.postinstall
-    system 'pip3 install --upgrade pip'
+    puts 'Updating pip packages...'.lightblue
+    @piplist = `pip list | awk '{print $1}'`.split
+    @piplist.drop(2).each do |pip_pkg|
```

Comment:
```
@piplist = `pip list | cut -d' ' -f1 | tail +3`.split
@piplist.each do |pip_pkg|
...
```
Removes need to drop.

Reply:
This also works:
```
@piplist = `pip list --format freeze | cut -d'=' -f1`.split
```

Reply:
I have the drop because using tail gives me this error:
```
chronos@cheekon-i686 /usr/local/lib/crew/packages $ crew postinstall python3
Updating pip packages...
tail: cannot open ‘+3’ for reading: No such file or directory
```

Reply:
This works though:
```
puts 'Updating pip packages...'.lightblue
    @piplist = `pip list | cut -d' ' -f1`.split
    @piplist.drop(2).each do |pip_pkg|
      system "pip install #{pip_pkg} -U", exception: false
    end
```